codecov.yml

```diff
@@ -59,3 +59,4 @@ coverage:
     patch:
       default:
         informational: true # Don't gate PRs based on Codecov passing thresholds
+        if_ci_failed: success
```

Comment:
this `# Don't gate PRs based on Codecov passing thresholds` wasn't actually working until the line i added